gcs/conn.go

```diff
@@ -183,8 +183,8 @@ func (c *conn) OpenBucket(
 	// Forbidden". Similarly for bad bucket names that don't collide with another
 	// bucket.
 	_, err = b.ListObjects(ctx, &ListObjectsRequest{MaxResults: 1})
-
-	if typed, ok := err.(*googleapi.Error); ok {
+	unwrapped := errors.Unwrap(err)
```

Comment:
what will happen if err is nil or unwrapped is nil?

Reply:
If err or unwrapped is nil, ok will be false so it will not go into that block. and bucket will successfully mount in the system.